docs/openfield.rst

```diff
@@ -0,0 +1,69 @@
+OpenField
```

Comment:
This looks great but don't forget to add a link to the connector to [the sidebar](https://www.parsonsproject.org/pub/build-a-new-connector/release/6#sidebar)